### DISCUSSION TITLE
Email template generated in parent theme not in child theme

### DISCUSSION THREAD

**Discussion by jijijou** [PrestaShop/PrestaShop#33399] [Ideas]
Hello, 

In case of using child theme, it's not possible to generate the email template in the child theme. Only in parent theme . It's would be great that we can choose the child theme into generator. 

@jolelievre , @mickaelandrieu has sent me your account for this issue ;) 

Thanks a lot

**Comment by khouloudbelguith** [comment]
Hi @jijijou,

Thanks for your report.
Ping @jolelievre what do you think?

Thanks!

**Comment by jf-viguier** [comment]
Yes, we need a parent/child system for email templating too.

**Comment by matks** [comment]
Ping @jolelievre wdyt? I think it would be great to allow mail themes to have parent/children system, to be able to benefit from parent theme and only customize what you need.

Do you think it would be doable?
